blocks/api/registration.js

```diff
@@ -168,7 +168,20 @@ export function registerBlockType( name, settings ) {
 		set( settings, [ 'supports', 'multiple' ], ! settings.useOnce );
 	}
 
-	dispatch( 'core/blocks' ).addBlockTypes( settings );
+	const implementationOnlyAttributes = [ 'transforms', 'edit', 'save', 'icon', 'getEditWrapperProps' ];
```

Comment:
`deprecated` is missing in here.

Reply:
Actually, it's not. I intentionally avoided any deprecations for now. The idea is to do it only once, when we have all the bits in (the server-side part essentially)

Reply:
I mean, it doesn't matter in this PR, but definitely something, we should keep in mind. Should we add todo comment?

Reply:
I think it's not necessary because that's the first thing we should do when we move registration to the server. Tell people to register their blocks on the server.